### PR TITLE
List of speakers UI rework

### LINKED ISSUE #3445: Fix layout in Autopilot
**Current behaviour:**
If the speaking times card is displayed on the Autopilot and a participant with number is moved on the LoS then the Layout changes while the participant is dragged around

**Reproduction:**
1. connect structure levels with countdowns
2. Add a number and structure level to participant A 
3. project sth with a LoS onto the Autopilot
4. add participant A to the LoS 
5. (requirement for the bug is at least one structure level in the speaking times card)
6. start dragging participant A on the LoS 
7. the distance from the autopilot content and the left side gets smaller
8. let go of participant A and stop dragging them
9. the distance jumps back to the original distance

Notes: the distance only changes if a participant with a number is dragged around

**Wanted behaviour:**
The distance should not change even if a participant with number is dragged on the LoS

### LINKED ISSUE #3459: List of Speakers: UI rework
**Description:** The list of speakers needs an UI rework to make it more accessible for tablet (mobile) users and to fit more the new structure level integration.

**What should be done:**
- [ ] UI rework like shown in image one. This affects the contributions and the buttons at the top of the list of speaker.
- [ ] From functionality perspective it's mostly the same. Just the layout, the positioning of the buttons and the metadata of a contribution should be changed. 
- [ ] One small change for contributions at the top of the list: If the contribution is at the top, the play-button should be highlighted in blue. The other Play-buttons should be in gray.

**How it should look like:**

**Image 1 - Desktop view:**
image

**Image 2 - Desktop view - Active contribution**
image

**Image 3 - tablet (mobile) view:**
image

**Image 4 - tablet (mobile) view - active contribution:**
image

### PR DESCRIPTION
**Pull request by bastianjoel**
resolves #3459
resolves #3445

Note that the layout of interposed questions looks a bit different if the browser does not support [`:has()`](https://caniuse.com/css-has).

### CODE REVIEW

**Review by Elblinator — COMMENTED** [review]
It looks rather nice and feels great to work with so far
two notes: 
1. can you please reduce the margin-top from .current-speaker to 10px? 
The 50px make it look very busy if you activate/deactivate current-speaker a lot
2. I have some other remarks and questions about the design. They should be rather simple but need some conceptual talk. I'll talk with @emanuelschuetze about it then get back to you

it really really cool in my opinion!

**Review by Elblinator — CHANGES_REQUESTED** [review]
in the list: 
- [x] .current-speaker { martin-top:10px; margin-bottom:0}
- [x] .speaker-name {font-weight: 500}
- [x] the **icons** from pro/contra should have the corresponding colour (green and red, like it was before)
- [x] PoO icon and text should be in the warn colour again
- <s>the text speaker-additional-info and speech-type should be rgba(0, 0, 0, 0.54);</s>
- [x] the middot should only be present if it is needed (add a participant withou any additional information and then activate the star contribution there will be a "icon Contribution middot"
- [x] the first interposed question should have a margin-top:0 and a boder of 1 px which should be transparent if the question is not active and it should be white if the question is active
- [x] every entry should have the same height, even if there is not enough information to fill three lines
- [x] the whole entry ( I think .speaker-info or .single-speaker-line) should have a padding top and bottom of 4px
- <s>please invert the filling from the stop button (see Mock-up)</s>
- [ ] the play buttons for every participant on the waiting list should be slightly smaller
- <s>the  play buttons for every participant on the waiting list (besides the first one) should be greyer (for now please use rgba(0, 0, 0, 0.54);)</s>
- [x] the play button for the first participant on the waiting list should be in accent colour
- [ ] UI for <ins>current</ins> speaker: 
   - [x] remove the "special colour" from pro/contra/PoO/contribution number 
   - [ ] the play button should be at the same placing as the other play buttons (meaning they should be directly under each other)
   - [ ] the os-countdown-time should be bigger
- [ ] Last speakers list: 
    - the word 'start time' should be removed
    - in front of the time should be the icon ' schedule' <s>
    - either 
A) pull the start time back into the os-list-of-speakers-entry (that would be better)
B) revert the visual changes from the list
-> the goal here is to make the list slimmer
other idea to reach that are also welcome</s>


this will be done in future issues: 
 <s>The title-line:
- if a LoS is closed add the lock icon (smaller) right next to the title in a raised manner  (for reference project a LoS onto a projector and close it)
- in the button which closes/opens the LoS: 
   -  make the Lock-icon as a permanent icon (the outlines should be white and the icon should not be filled
   - [x] change the tooltips to "Close list of speakers" and "Open list of speakers"
- change the currently used icon for the re-add button to the "undo" icon</s>

**Comment by bastianjoel** [comment]
> the play buttons for every participant on the waiting list (besides the first one) should be greyer (for now please use rgba(0, 0, 0, 0.54);)
I don't like this. I would prefer sticking to the colors available by angular material

> the play buttons for every participant on the waiting list should be slightly smaller
Please specify how small exactly. Same as stop button?

> please invert the filling from the stop button (see Mock-up)
Material design icons is lacking a outlined stop button

> Last speakers list
Directly following speakers are currently categorized. I don't like the old design and I would really like to get this unified into the speaker entry. For option A on the other hand we are lacking space there.

**Comment by bastianjoel** [comment]
> change the currently used icon for the re-add button to the "undo" icon

It is already the undo icon?

**Comment by Elblinator** [comment]
I edited my change request accordingly to your comments and the discussion

**Review by emanuelschuetze — APPROVED** [review]
Looks good now!

File: client/src/app/site/pages/meetings/modules/list-of-speakers-content/components/list-of-speakers-entry/list-of-speakers-entry.component.scss [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+            width: $speaker-control-button-font-size;
+            height: $speaker-control-button-font-size;
+        }
+    }
+
+    .mat-mdc-button-touch-target {
+        width: $speaker-control-button-size !important;
+        height: $speaker-control-button-size !important;
+    }
+    */
```
Comment (luisa-beerboom) [inline client/src/app/site/pages/meetings/modules/list-of-speakers-content/components/list-of-speakers-entry/list-of-speakers-entry.component.scss:136]: ?

File: client/src/app/site/pages/meetings/modules/list-of-speakers-content/components/list-of-speakers-entry/list-of-speakers-entry.component.scss [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
```
Comment (luisa-beerboom) [inline client/src/app/site/pages/meetings/modules/list-of-speakers-content/components/list-of-speakers-entry/list-of-speakers-entry.component.scss:179]: See above

**Review by luisa-beerboom — APPROVED** [review]

**Pull request merged**